### PR TITLE
Add new Braze newsletter banner

### PR DESCRIPTION
**Pull request by KaliedaRik**
## What does this change?
This PR:
+ Adds a small backend change to get DCR to send a function to Braze which Braze banners can then use for single-click newsletter signup - this replicates existing functionality for Braze epic messages.
+ Updates Storybook with the new Braze Newsletter Banner, and removes an unused banner from Storybook
+ Bumps the version of the `braze-components` dependency to latest

## Screenshots
Nothing interesting to show here

### CODE REVIEW

File: dotcom-rendering/src/web/components/StickyBottomBanner.importable.tsx [hunk 1]
```diff
@@ -196,7 +196,8 @@ const buildBrazeBanner = (
 	candidate: {
 		id: 'braze-banner',
 		canShow: () => canShowBrazeBanner(brazeMessages, brazeArticleContext),
-		show: (meta: any) => () => <BrazeBanner meta={meta} />,
+		show: (meta: any, idApiUrl?: string) => () =>
```
[the remark this answers is not in the document]
Comment (KaliedaRik) [reply dotcom-rendering/src/web/components/StickyBottomBanner.importable.tsx:199]: I'm very happy to give this a go!

[the remark this answers is not in the document]
Comment (KaliedaRik) [reply dotcom-rendering/src/web/components/StickyBottomBanner.importable.tsx]: ... done

**Comment by KaliedaRik** [comment]
The current typescript errors are expected and should resolve once we update the braze-component dependency to match latest braze-component version following the merge of this BC branch into production https://github.com/guardian/braze-components/pull/394

**Review by tomrf1 — APPROVED** [review]
👍

**Pull request merged**